FLIMfitFrontEnd/Classes/@flim_data_series/flim_data_series.m

```diff
@@ -601,14 +601,7 @@ function estimate_g_factor(obj)
        
         function set.data_size(obj,data_size)
             s = length(data_size);
-            if s == 3
-                % not sure wht this check is needed
-                % but allow data_size to be 3 for 1xn images
-                if data_size(2) > 1
-                    data_size = [data_size(1) 1 data_size(2:3)];
-                end
-            end
-            obj.data_size = [data_size(:) ; ones(5-s,1)];
+            obj.data_size = [data_size(:); ones(5-s,1)];

```

Comment:
Are we confident that this check is no longer needed? I wondered if some "stack of tiffs" format might still need it.